packages/ui-gui/package.json

```diff
@@ -1,28 +1,38 @@
 {
   "name": "@webpack-cli/ui-gui",
   "version": "0.0.1",
+  "license": "MIT",
   "description": "GUI for webpack-cli ui",
-  "main": "index.js",
+  "main": "src/index.js",
   "scripts": {
-    "build": "webpack --config=webpack.prod.js",
-    "serve": "webpack-dev-server --config=webpack.prod.js"
+    "start": "if-env NODE_ENV=production && npm run -s serve || npm run -s dev",
+    "build": "preact build",
+    "serve": "preact build && preact serve",
+    "dev": "preact watch",
+    "lint": "eslint src"
   },
-  "license": "MIT",
+  "eslintConfig": {
+    "extends": "eslint-config-synacor",
+    "rules": {
+      "no-unused-vars": "warn",
+      "react/sort-comp": "off",
+      "lines-around-comment": "off",
+      "react/prefer-stateless-function": "off"
+    }
+  },
+  "eslintIgnore": [
+    "build/*"
+  ],
   "devDependencies": {
-    "@babel/core": "^7.1.2",
-    "@webpack-cli/init": "^0.1.2",
-    "babel-core": "^6.26.3",
-    "babel-loader": "^8.0.4",
-    "babel-plugin-syntax-dynamic-import": "^6.18.0",
-    "babel-preset-env": "^1.7.0",
-    "css-loader": "^1.0.0",
-    "mini-css-extract-plugin": "^0.4.4",
-    "node-sass": "^4.9.3",
-    "sass-loader": "^7.1.0",
-    "style-loader": "^0.23.1",
-    "webpack": "^4.21.0",
-    "webpack-cli": "^3.1.2",
-    "webpack-dev-server": "^3.1.10"
+    "eslint": "^4.5.0",
+    "eslint-config-synacor": "^1.1.0",
+    "if-env": "^1.0.0",
+    "preact-cli": "^2.0.0"
   },
-  "dependencies": {}
+  "dependencies": {
+    "preact": "^8.2.1",
+    "preact-compat": "^3.17.0",
```

Comment:
Compact is not needed if we don't use react

Reply:
@ematipico 👍